airbyte-integrations/connectors/source-s3/unit_tests/test_s3file.py

```diff
@@ -15,24 +15,6 @@
 
 
 class TestS3File:
-    @pytest.mark.parametrize(  # passing in full provider to emulate real usage (dummy values are unused by func)
-        "provider, return_true",
-        [
-            ({"storage": "S3", "bucket": "dummy", "aws_access_key_id": "id", "aws_secret_access_key": "key", "path_prefix": "dummy"}, True),
```

Comment:
Why were these tests removed?

Reply:
I removed it because we don't have `use_aws_account` function anymore. Instead the logic was moved to `make_s3_client`. I am not quite sure how to add a unit test for that though. Open to suggestions.